documentation/developer/getting_started/03_syntax.md

```diff
@@ -17,7 +18,7 @@ In a future release, Leo will support other sections to give information on `[de
 [package]
 ```  
 
-The package `name` is the official, unique name of the package that other developers will be able to look up after you have published your package.
+The package `name` is the official name of the package that other developers will be able to look up after you have published your package.
```

Comment:
We provide uniqueness by summing `author` and `name` therefore package name itself is not unique (multiple hello-worlds on Aleo PM prove it).